spec/features/users_can_edit_user_info_spec.rb

```diff
@@ -140,6 +170,18 @@
     end
   end
 
+
+  scenario "cannot update while not logged in" do
+    allow_any_instance_of(ApplicationController)
+    .to receive(:current_user).and_return(nil)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.